### ISSUE TITLE
Request: Menu bar

### ISSUE THREAD

**Issue by Nomanpour** [TomSchimansky/CustomTkinter#220]
Hello,

Thanks for your great job!

Is it possible to add support for the Tkinter.Menu?

Thank you in advance.

**Comment by Nomanpour** [comment]
I found something in the widgets directory but I couldn't find the documentation about how to use it.

**Comment by felipetesc** [comment]
I'll create a sample and post it on ctk prototype non official doc. When I finish I send the link .

**Comment by felipetesc** [comment]
> Tkinter.Menu

Are you referring to dropdown menu ?

**Comment by Nomanpour** [comment]
Yes, thanks.

**Comment by felipetesc** [comment]
This is a simple sample, the menu should be worked more. For now I hope it can help you:

The code below can be added to your project, please test it to see what's wrong, and what do you dislike, so I can change it.

```python
import tkinter
import customtkinter

from customtkinter import ThemeManager
from customtkinter import AppearanceModeTracker
from customtkinter import ScalingTracker
from customtkinter import CTkBaseClass


class CTkMenu(tkinter.Menu, CTkBaseClass):

    def get_color_from_name(self, mode, name: str):
        color = ThemeManager.theme["color"][name][mode]
        #ThemeManager.single_color(color, mode)
        return color

    def add_cascade(self, cnf={}, **kw):
        self.add('cascade', cnf or kw)

    def __init__(self, master: tkinter.Tk, text_font = "default_theme", cnf={}, **kw):
        super().__init__()
        self.option_add('*tearOff', False)
        
        self.mode = AppearanceModeTracker.get_mode()
        ctk_bg =  self.get_color_from_name(self.mode, "frame_high")
        ctk_fg = self.get_color_from_name(self.mode, "text")
        ctk_hover_bg = self.get_color_from_name(self.mode, "button")
        self.text_font = (ThemeManager.theme["text"]["font"], ThemeManager.theme["text"]["size"]) if text_font == "default_theme" else text_font
        #self.font = CTkBaseClass.apply_font_scaling(self.text_font)
        self.configure(background=ctk_bg)
        self.configure(foreground=ctk_fg)
        self.configure(borderwidth=0)
        self.configure(activebackground=ctk_hover_bg)
        self.config(font=self.text_font)
```
And the code below is a simple test : 
```python

import customtkinter
from menu import CTkMenu

DARK_MODE = "dark"
customtkinter.set_appearance_mode(DARK_MODE)
customtkinter.set_default_color_theme("blue")

def donothing():
   filewin = Toplevel(root)
   button = Button(filewin, text="Do nothing button")
   button.pack()

def set_right_pos(win):
    max_width = int(win.winfo_screenwidth() / 2)
    max_height = int(win.winfo_screenheight() / 2)
    x = max_width
    y = 0
    right_pos = str(max_width) + "x" + str(max_height) + "+"\
                + str(x) + "+" + str(y)
    print(right_pos)
    return right_pos

class App(customtkinter.CTk):

    color = "dark"

    def on_click(self):

        pass

    def __init__(self):
        super().__init__()
        # self.state('withdraw')
        self.title("Modified Menubar")
        self.geometry(set_right_pos(self))

        menu = CTkMenu(self)

        self.config(menu=menu)

        fileMenu = CTkMenu(menu)
        fileMenu.add_command(label="Item")
        fileMenu.add_command(label="Exit", command=self.exitProgram)
        menu.add_cascade(label="File", menu=fileMenu)

        editMenu = CTkMenu(menu)
        editMenu.add_command(label="Undo")
        editMenu.add_command(label="Redo")
        menu.add_cascade(label="Edit", menu=editMenu)

    def exitProgram(self):
        exit()

a = App()
a.mainloop()
```
![peek_menu](https://user-images.githubusercontent.com/2336812/177012825-abd8e0ce-2664-43da-8989-f6575a38ece5.gif)

**Comment by Nomanpour** [comment]
Thanks for sharing, I will try it

**Comment by RunsOnRum** [comment]
Tom, I am fairly new to python and have attempted to implement this code to generate a menubar. I have managed to compile but only after I commented out the self.configure.... lines in the class as they throw a ValueError "background" are not supported arguments. The error is the same for foreground, borderwidth, etc.

I get a different error with self.config(font=self.text_font). AttributeError 'config' is not implemented for CTk widgets.

For the sake of learning more, I decided to ask the question here. I appreciate your work and a beautiful looking implementation of tkinter.
